cypress/support/testHelper.js

```diff
@@ -25,6 +25,30 @@ export const checkElementStyles = (elementString, text, color, fontFamily) => {
   shouldContainStyles(el, 'font-family', fontFamily);
 };
 
+export const checkFooterLinks = (position, url) => {
```

Comment:
These test helpers are quite ambiguous, and could be namespaced to footer helpers.